addons/web_editor/static/src/js/editor/odoo-editor/test/spec/insertHTML.test.js

```diff
@@ -90,6 +93,20 @@ describe('insert HTML', () => {
                 contentAfter: '<pre>abcdef[]<br>ghi</pre>',
             });
         });
+        it('should unpack elements from a lone paragraph (Knowledge check)', async () => {
+            // TODO remove this test if/when the execCommand ends up not unpacking content of a
+            // paragraph or from the first and last inserted elements (in case there are more than
+            // one). Don't forget to update the onPreRendered callback in updateBehaviors in
+            // Knowledge at that point.
+            await testEditor(BasicEditor, {
+                contentBefore: '<p>[]<br></p>',
+                stepFunction: async editor => {
+                    setCursorEnd(editor.editable);
+                    await editor.execCommand('insert', parseHTML('<p><div>content</div><p><br></p></p>'));
+                },
+                contentAfter: '<p><br></p><div>content</div><p><br>[]</p>',
+            });
+        });
```

Comment:
That is in fact a bug, I think we discussed about it already. Maybe I did not create a task ? Or we just haven't done it yet ? I need to check.